tests/unit/htdocs/class/CriteriaTest.php

```
@@ -0,0 +1,1829 @@
+        $this->assertStringStartsWith('`p` LIKE ', $result);
+    }
+
+    // =========================================================================
+    // Criteria — Value types
+    // =========================================================================
+
+    public function testRenderBoolTrueValueIsQuotedAsString(): void
+    {
+        // bool true: is_int(true) = false, is_string(true) = false
+        // so it falls to quote((string)true) = quote("1") = '1'
+        $c = new Criteria('active', true);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $c. Configured minimum length is 3. (undefined)

(ShortVariable)